README.md

```diff
@@ -61,6 +61,7 @@ style. The following options are settable through a `data-` property on the
 * `accept-on-scroll` - when is set `true` window scrolling is considered as acceptance. (default: `false`)
 * `accept-on-click` - when is set `true` any click on the page is considered as acceptance. (default: `false`)
 * `accept-on-timeout` - when is set to an integer value (milliseconds) it automatically agree and close the banner after the N milliseconds selected. (default: null)
```

Comment:
I think this reads better:

```
Automatically agrees and closes the notice after specified number of milliseconds. (default: `null`)
```

But I might take a stab at changing a bunch of these anyway in the next few days anyway, so if you don't feel like changing anything, leave them as is.

Thanks a bunch for the PRs!